scripts/operations/node_management/ncn-ims-image-upload.sh

```diff
@@ -122,4 +136,16 @@ cray ims images update "$IMS_IMAGE_ID" \
         --link-etag "${MANIFEST_ETAG}" \
         --link-path "s3://boot-images/$IMS_IMAGE_ID/manifest.json" > /dev/null
 
+podman run --rm --name ncn-cpc \
+    --user root \
+    -e PRODUCT=csm \
+    -e PRODUCT_VERSION=$CSM_RELEASE \
+    -e YAML_CONTENT_STRING="{images: {$IMS_IMAGE_NAME: {id: $IMS_IMAGE_ID}}}" \
+    -e KUBECONFIG=/.kube/admin.conf \
+    -e VALIDATE_SCHEMA="true" \
+    -e SET_ACTIVE_VERSION="true" \
```

Comment:
I'll suggest that we not use the `SET_ACTIVE_VERSION` flag here. I don't think we would want to be changing what version of csm is "active" at this point.

We've decided to move away from using this flag entirely (lots of context found in: https://jira-pro.its.hpecorp.net:8443/browse/CRAYSAT-1629) but in any case I don't think we would want the value changing just as a part of uploading the ims images.